test/e2e/usersignup_test.go

```diff
@@ -466,6 +466,85 @@ func (s *userSignupIntegrationTest) TestTransformUsername() {
 	}
 }
 
+func (s *userSignupIntegrationTest) TestReturningUserProvisionedToLastCluster() {
+	hostAwait := s.Host()
+	memberAwait := s.Member1()
+	memberAwait2 := s.Member2()
+
+	s.T().Run("test a single user activated then deactivated then reactivated and verify provisioned to same cluster both times", func(t *testing.T) {
+		// given
+		hostAwait.UpdateToolchainConfig(testconfig.AutomaticApproval().Enabled(true))
+
+		// when
+		userSignup, _ := s.newSignupRequest().
+			Username("returninguser").
+			Email("returninguser@redhat.com").
+			EnsureMUR().
+			RequireConditions(ConditionSet(Default(), ApprovedAutomatically())...).
+			Execute().Resources()
+
+		// then
+		// Confirm the MUR was created and target cluster was set
+		VerifyResourcesProvisionedForSignup(s.T(), s.Awaitilities, userSignup, "base")
+
+		s.T().Run("user activated -> deactivated -> reactivated", func(t *testing.T) {
+			// given
+			mur, err := hostAwait.WaitForMasterUserRecord(userSignup.Status.CompliantUsername, wait.UntilMasterUserRecordHasConditions(Provisioned(), ProvisionedNotificationCRCreated()))
+			require.NoError(t, err)
```

Comment:
the `s.newSignupRequest()....Resources()` returns MUR as the second value - you can use that one instead of getting it here again.